### PR TITLE
feat: only username, email, password, date_of_birth, height, weight, goals are required now

### PR DESCRIPTION
**Pull request by Eowiin**
## Description

Changes for backend authentification for user's registration.

## Changement

Removal of all fields except those in the title for user's registration.
Adaptation of the tests to work with these changes.

## Commentaire

In accordance with the last group review, only essential fields are required to create a new user in database. Tests have been updated to work with these changes.

### CODE REVIEW

File: backend/playfit/authentification/models.py
Comment (Masteruuuuu) [inline backend/playfit/authentification/models.py:1]: Should we keep the first name and last name optional ? Or just delete them ?

**Review by Masteruuuuu — COMMENTED** [review]
@christophechr need your opinion if possible

File: backend/playfit/authentification/admin.py
Comment (christophechr) [inline backend/playfit/authentification/admin.py:1]: No need for first name and last name now, I'll make sure to emphasize the RGPD rules around the end of the next week.

File: backend/playfit/authentification/migrations/0004_alter_customuser_first_name_and_more.py
Comment (christophechr) [inline backend/playfit/authentification/migrations/0004_alter_customuser_first_name_and_more.py:1]: Same here

File: backend/playfit/authentification/models.py
Comment (christophechr) [reply backend/playfit/authentification/models.py:1]: We can delete them for now

**Review by christophechr — CHANGES_REQUESTED** [review]

**Review by christophechr — APPROVED** [review]

**Review by Masteruuuuu — APPROVED** [review]

**Review by Masteruuuuu — APPROVED** [review]

**Pull request merged**